Tools/Scripts/build-and-collect-pgo-profiles

```diff
@@ -124,15 +134,18 @@ SPTH='OpenSource/Tools/Scripts'
 $SPTH/run-benchmark "${jsargs[@]}"
 $SPTH/pgo-profile merge "$BASE/jetstream"
 
-$SPTH/run-benchmark "${spargs[@]}"
-$SPTH/pgo-profile merge "$BASE/speedometer"
+$SPTH/run-benchmark "${sp2args[@]}"
+$SPTH/pgo-profile merge "$BASE/speedometer2"
+
+$SPTH/run-benchmark "${sp3args[@]}"
+$SPTH/pgo-profile merge "$BASE/speedometer3"
 
 $SPTH/run-benchmark "${mmargs[@]}"
 $SPTH/pgo-profile merge "$BASE/motionmark"
 
 rm *.result
 
-$SPTH/pgo-profile combine --jetstream "$BASE/jetstream" --speedometer "$BASE/speedometer" --motionmark "$BASE/motionmark" --output "$BASE/output"
+$SPTH/pgo-profile combine --jetstream "$BASE/jetstream" --speedometer2 "$BASE/speedometer2" ---speedometer3 "$BASE/speedometer3" --motionmark "$BASE/motionmark" --output "$BASE/output"
```

Comment:
Is `---speedometer3` (3 dashes) intentional?

Reply:
Nice catch. I've updated the patch accordingly.